sklearn/neighbors/_base.py

```diff
@@ -315,8 +315,7 @@ def __init__(self, n_neighbors=None, radius=None,
         self._check_algorithm_metric()
 
     def _check_algorithm_metric(self):
-        if self.algorithm not in ['auto', 'brute',
-                                  'kd_tree', 'ball_tree']:
+        if self.algorithm not in ['auto', 'brute', 'kd_tree', 'ball_tree']:
```

Comment:
We try to avoid these cosmetic changes in code. It makes `git blame` a little easier to use. May you revert this?

Reply:
Reverted.